packages/search-token-lambda/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@coveo/search-token-lambda",
+  "version": "1.19.0",
+  "private": "true",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:coveo/cli.git",
+    "directory": "packages/search-token-lambda"
+  },
+  "scripts": {
+    "start:dev": "netlify functions:serve",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "npm:bump:template": "npm --no-git-tag-version version",
+    "npm:publish:template": "npm publish --access public --tag latest --registry http://localhost:4873"
```

Comment:
If you do this, you need to add it to the lerna config.

Reply:
for lerna everything under ./packages is a package, right?

Reply:
Yes! I meant verdaccio, corrected.